recipes/tardigrade_abaqus_tools/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "tardigrade_abaqus_tools" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCBoulder/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: cebcdd3905b04bd233aadc73189391b787f7cb9dd47100e4b8de8d6a4f0306ec
+
+build:
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script:
+    - set -x
+    # Build and install c++ libraries
+    - cmake -S . -B build -DCMAKE_BUILD_TYPE=Release ${CMAKE_ARGS}
+    - cmake --install build --prefix ${PREFIX}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # TODO: Remove when the upstream project updates the CMakeLists.txt file
+    - {{ stdlib('c') }}  # TODO: Remove when the upstream project updates the CMakeLists.txt file
```

```suggestion
    - {{ stdlib('c') }}
```

even if the project just requires a C++ compiler, you still need `stdlib('c')`.